client/public/index.html

```diff
@@ -5,6 +5,12 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+
+    <!--Import icons, fonts, Materialize style-->
+    <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet">
+	  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link type="text/css" rel="stylesheet" href="./assets/css/materialize.css"  media="screen,projection"/>
+    <link rel="stylesheet" href="./assets/css/style.css">
```

Comment:
Going the old-fashioned route with Materialize. Material-ui, react-bootstrap, reactstrap proved difficult and maybe unnecessary